server/adaptors/integrations/__data__/repository/aws_vpc_flow/assets/vpc_agg_30min_connections_mv-1.0.0.sql

```diff
@@ -0,0 +1,37 @@
+-- 30 Min Aggregation MV of VPC connections / bytes / packets
```

Comment:
I think the file gets flattened at some point of install, so comments will cause issues